Cargo.toml

```diff
@@ -12,16 +12,16 @@ version = "20.3.3"
 rust-version = "1.74.0"
 
 [workspace.dependencies.soroban-env-host]
-version = "=20.2.1"
+version = "=20.2.2"
 # git = "https://github.com/stellar/rs-soroban-env"
 # rev = "1bfc0f2a2ee134efc1e1b0d5270281d0cba61c2e"
-# path = "../rs-soroban-env/soroban-env-host"
+path = "../rs-soroban-env/soroban-env-host"
 
 [workspace.dependencies.soroban-simulation]
-version = "=20.2.1"
+version = "=20.2.2"
 # git = "https://github.com/stellar/rs-soroban-env"
 # rev = "1bfc0f2a2ee134efc1e1b0d5270281d0cba61c2e"
-# path = "../rs-soroban-env/soroban-simulation"
+path = "../rs-soroban-env/soroban-simulation"
```

Comment:
Integration tests won't work when pointing a local directory which doesn't exist in CI

Reply:
Sure, these are just my local changes. There was no env revision yet at the time of posting this.